server/src/components.yaml

```diff
@@ -0,0 +1,74 @@
+components:
+  schemas:
+    NewFrosh:
+      type: object
+      required:
+        - name
```

Comment:
Wouldn't this be email? I guess we can figure this out after registration gets back to us though

Reply:
I'm currently planning on figuring out what is required after registration gets back to us, for now this is more of a proof of concept that we can have required parameters